Instagram/View Controllers/HomeViewController.m

```diff
@@ -27,6 +29,10 @@ - (void)viewDidLoad {
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
     
+    self.refreshControl = [UIRefreshControl new];
+    [self.refreshControl addTarget:self action:@selector(refreshData:) forControlEvents:UIControlEventValueChanged];
+    [self.tableView insertSubview:self.refreshControl atIndex:0];
```

Comment:
nit: I would put this in a separate method just for tidiness. `configureRefreshControl` or something.